src/App.jsx

```diff
@@ -139,6 +162,25 @@ const App = () => {
     fetchData();
   }, []);
 
+  function sortTodos(todos) {
+    return todos.sort((objectA, objectB) => {
+      if (sortAsc) {
+        return sortTodosAscending(objectA.title, objectB.title)
+      } else {
+        return sortTodosDescending(objectA.title, objectB.title)
+      }
+    });
+  }
+
+  function handleSortToggleClick() {
```

Comment:
is there a better, more intuitive name for this function we can use? Perhaps something like `handleToggleSort`, `handleSortToggle`, or `handleSortClick`?